src/main/scala/com/pingcap/tispark/TiRDD.scala

```diff
@@ -48,11 +50,11 @@ class TiRDD(val selectReq: TiSelectRequest,
   }
 
   override def compute(split: Partition, context: TaskContext): Iterator[Row] = new Iterator[Row] {
-    selectReq.bind
+    selectReq.resolve()
     // bypass, sum return a long type
-    val tiPartition = split.asInstanceOf[TiPartition]
-    val iterator = snapshot.select(selectReq, split.asInstanceOf[TiPartition].task)
-    val finalTypes = rowTransformer.getTypes.toList
+    val tiPartition: TiPartition = split.asInstanceOf[TiPartition]
+    val iterator: util.Iterator[TiRow] = snapshot.tableRead(selectReq, split.asInstanceOf[TiPartition].task)
```

Comment:
I don't think `Iterator` here stands for that under `java.util` package...

Reply:
It actually is. You can check signature of tableRead. It returns a `SelectIterator` which implements `util.Iterator`.